README.rst

```diff
@@ -339,6 +339,17 @@ Choose from the list of available rules:
 
   *Risky rule: forcing strict types will stop non strict code from working.*
 
+* **declare_strict_types_position**
+
+  Move the strict type declaration to the configured location. Requires
+  PHP >= 7.0.
+
+  Configuration options:
+
+  - ``relocate_to`` (``'next'``, ``'same'``): whether ```declare(strict_types=1)````` should
```

Comment:
go ahead and sent a fix. in the meantime, fix to single backticks to not crap the readme